Azure Service Hunting/Azure Kubernetes Service Guided Hunting.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    let _endTime = datetime('2022-04-26T00:00');
</pre><p>You might want to make these relative times like the other queries.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Good catch @petebryan , I thought I caught all of those! I've fixed that and also updated the `_startTime` and `_endTime` variables to be in a consistent format so that the time range filtering statement for all the KQL queries are the same!